jetty-ee11/jetty-ee11-websocket/jetty-ee11-websocket-jakarta-client/src/main/java/org/eclipse/jetty/ee11/websocket/jakarta/client/JakartaWebSocketClientContainer.java

```diff
@@ -207,6 +195,13 @@ private Session connect(ConfiguredEndpoint configuredEndpoint, URI destURI) thro
                 throw (DeploymentException)cause;
             if (cause instanceof IOException)
                 throw (IOException)cause;
+            if (cause instanceof UpgradeException)
```

Comment:
There were a lot of WebSocket changes made only to EE11 to pass the websocket TCK.

I only made these in EE11 because some of them are potentially breaking changes to the user. 

So we could backport it to EE10 and potentially break people upgrading to `12.1.x` (then we would pass websocket TCK on both EE10 and EE11). OR we not backport and preserve backward compatibility with Jetty 12.0 EE10 websocket.